config/vcrhelixconfig/CorpVcrClusterCreationHelixConfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I would suggest renaming this to something other than corp since this is our open source codebase. Could you also explain why the parameters differ for this use case?

Reply:
I guess I can remove this from here (and maybe check it in the closed source code?). The reason some parameters differ here is because the number of nodes are different in corp cluster.